disable Session Replay document correction (#13555)
DataDog/documentation: content/en/real_user_monitoring/session_replay/_index.md

- Remove startSessionReplayRecording() to stop session recordings.
- Set `replaySampleRate` to `0` to stop collecting the RUM Session Replay plan which includes resources and long tasks.

> - Upgrade your Browser RUM SDK to version >= 3.6 for these configuration options to apply.

Reviewer: ```suggestion

In order to apply these configurations, upgrade the Browser RUM SDK to a version >= 3.6.
```
